tests/converters/table-cell-paragraph-post-fixer.js

```diff
@@ -221,4 +229,58 @@ describe( 'Table cell paragraph post-fixer', () => {
 			'</table>'
 		) );
 	} );
+
+	it( 'should not be run on changing attribute of other element in a table cell', () => {
+		setModelData( model,
+			'<table>' +
+				'<tableRow>' +
+					'<tableCell><block></block></tableCell>' +
+				'</tableRow>' +
+			'</table>'
+		);
+
+		postFixerSpy.resetHistory();
+
+		// Insert table row with one table cell
+		model.change( writer => {
+			writer.setAttribute( 'foo', 'bar', root.getNodeByPath( [ 0, 0, 0, 0 ] ) );
+		} );
+
+		expect( formatTable( getModelData( model, { withoutSelection: true } ) ) ).to.equal( formatTable(
+			'<table>' +
+				'<tableRow>' +
+					'<tableCell><block foo="bar"></block></tableCell>' +
+				'</tableRow>' +
+			'</table>'
+		) );
+
+		sinon.assert.calledOnce( postFixerSpy );
```

Comment:
Should "not be run" but `.calledOnce()`?

Reply:
Ugh, okay, this is convoluted :|. Called once means that no other post-fixer was called. I wonder if we can spy this in a better way (`Differ#refreshItem` might be a little less bullet-proof though). :(

Reply:
OTOH, `calledTwice` is not bullet-proof either.

Reply:
I'll check, I forgot about spying on differ :man_facepalming: